src/components/Editor/Tab.js

```diff
@@ -156,11 +156,31 @@ class Tab extends PureComponent<Props> {
     const isPrettyCode = isPretty(source);
     const sourceAnnotation = getSourceAnnotation(source, sourceMetaData);
 
-    function onClickClose(ev) {
-      ev.stopPropagation();
+    function onClickClose(e) {
+      e.stopPropagation();
       closeTab(source.get("url"));
     }
 
+    function handleTabClick(e) {
```

Comment:
I think we can shorten this function up quite a bit:

```js
e.preventDefault();
e.stopPropagation();

if(e.button === 1) {
    return closeTab(source.get("url"));
}

return selectSource(sourceId);
```

Reply:
Yeah, definitely.